src/main/java/org/jabref/gui/actions/ActionHelper.java

```diff
@@ -26,6 +27,13 @@ public static BooleanExpression needsDatabase(StateManager stateManager) {
         return stateManager.activeDatabaseProperty().isPresent();
     }
 
+    public static BooleanExpression needsSharedDatabase(StateManager stateManager){
+        var binding = Bindings.createBooleanBinding(() ->
+                        stateManager.activeDatabaseProperty().getValue().filter(context -> context.getLocation() == DatabaseLocation.SHARED).isPresent(),
```

Comment:
Please use `Easybind.map(stateManager.activeDatabaseProperty(), ...)` instead of `createBooleanBinding`. It's a bit more readable and shorter.